manifests/profile/puppetdb.pp

```diff
@@ -28,18 +31,39 @@
   Puppet_metrics_dashboard::Puppetdb_metric $puppetdb_metrics = puppet_metrics_dashboard::puppetdb_metrics(),
   Integer[1] $port                                            = 8081,
   String[2] $interval                                         = '5s',
+  Boolean $enable_client_cert                                 = true,
   ){
 
+  if ! defined(Puppet_metrics_dashboard::Certs['telegraf']) {
```

Comment:
I suggest using stdlib's ensure_resource for this

Reply:
Great idea! I changed both locations it is used.